dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -34,6 +34,17 @@ def post_ui_tip_dismissed
     render status: 200, json: {set: property_name}
   end
 
+  # POST /api/v1/users/<user_id>/accept_data_transfer_agreement
+  def accept_data_transfer_agreement
+    @user.properties[:data_transfer_agreement_accepted] = true
```

Comment:
Can we just refer to `@user.data_transfer_agreement_accepted`?

Reply:
we sure can. updated!